changelog.d/20250326_102514_PL-133259_disable-ocsp-config_scriv.md

```diff
@@ -0,0 +1,19 @@
+<!--
+
+A new changelog entry.
+
+Delete placeholder items that do not apply. Empty sections will be removed
+automatically during release.
+
+Leave the XX.XX as is: this is a placeholder and will be automatically filled
+correctly during the release and helps when backporting over multiple platform
+branches.
+
+-->
+
+### Impact
+
+
+### NixOS XX.XX platform
+
+- [PL-133259] Remove SSL Stapling from the default Nginx configuration since the default CA for NixOS provisioned certificates (Let's Encrypt) is ending OCSP support in 2025
```

Comment:
Please use parentheses at the end, when referencing tickets.

Reply:
The style in [our last changelog](https://doc.flyingcircus.io/platform/changes/2025/r008.html) was using *round brackets*, not *angle brackets*.
I am okay with fixing such a small nitpick only later during the final release review of the changelog.